Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1375,8 +1375,7 @@ ExceptionOr<void> HTMLElement::showPopover()
     popoverData()->setPreviouslyFocusedElement(nullptr);
 
     Style::PseudoClassChangeInvalidation styleInvalidation(*this, {
-        { CSSSelector::PseudoClassOpen, true },
-        { CSSSelector::PseudoClassClosed, false }
+        { CSSSelector::PseudoClassPopoverOpen, true }
```

Comment:
`Style::PseudoClassChangeInvalidation styleInvalidation(*this, CSSSelector::PseudoClassPopoverOpen, true);`